IdentityCore/src/network/error_handler/MSIDAADRequestErrorHandler.m

```diff
@@ -41,17 +41,28 @@ - (void)handleError:(NSError *)error
             context:(id<MSIDRequestContext>)context
     completionBlock:(MSIDHttpRequestDidCompleteBlock)completionBlock
 {
-    if (!httpResponse)
+    BOOL shouldRetry = YES;
+    shouldRetry &= httpRequest.retryCounter > 0;
```

Comment:
@jasoncoolmax Please help with this part as it touched your previous work. Thanks!

Reply:
can we simplify it to be ` BOOL shouldRetry =  httpRequest.retryCounter > 0;` ?